### DISCUSSION TITLE
11 Oct Battle Notes

### DISCUSSION THREAD

**Discussion by pinheadmz** [bitcoin-dev-project/warnet#648] [General]
Any issues, bugs, poor documentation, complaints, suggestions or compliments can go here.

**Comment by pinheadmz** [comment]
kubernetes needs to be explicitly turned on in docker desktop

**Reply by pinheadmz** [reply]
https://github.com/bitcoin-dev-project/warnet/pull/650

**Comment by pinheadmz** [comment]
> Release "ingress-nginx" does not exist. Installing it now.
> Error: Unable to continue with install: ClusterRole "ingress-nginx" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "ingress-nginx"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "ingress"

**Comment by pinheadmz** [comment]
warnet run vs warnet deploy... maybe they can catch user mixup if arg is a file not a direcotry and reccomend the other command

**Reply by pinheadmz** [reply]
https://github.com/bitcoin-dev-project/warnet/pull/652

**Comment by edilmedeiros** [comment]
Add 5k inv bug to the blog.

**Reply by pinheadmz** [reply]
@m3dwards @willcl-ark

**Comment by pinheadmz** [comment]
Red team was hardcoded in my_first_attack so either red team ran it or someone didn't change it...

**Reply by pinheadmz** [reply]
https://github.com/bitcoin-dev-project/battle-of-galen-erso/commit/916a429bbf6a050c47bd2d4c0070f824c97b26e1

**Comment by pinheadmz** [comment]
`warnet logs -f` stops after some volume of log lines and needs to be restarted

**Reply by willcl-ark** [reply]
Odd one

**Reply by pinheadmz** [reply]
`read_namespaced_pod_log()` does accept an optional `limit_bytes` argument but there is no default so there should be no limit...

**Reply by pinheadmz** [reply]
https://github.com/kubernetes-client/python/issues/1546

closed without fix

**Comment by pinheadmz** [comment]
add to docs: `warnet run ... -- --help` to get args for a scenario

**Reply by pinheadmz** [reply]
https://github.com/bitcoin-dev-project/battle-of-galen-erso/commit/916a429bbf6a050c47bd2d4c0070f824c97b26e1

**Comment by pinheadmz** [comment]
bump: https://github.com/bitcoin-dev-project/warnet/issues/540

**Reply by pinheadmz** [reply]
https://github.com/bitcoin-dev-project/warnet/pull/651

**Comment by pinheadmz** [comment]
We should also make `MiniWallet` easier to use in scenarios, maybe we can include an example where we call `MiniWallet.create_self_transfer()` where the argument `utxo_to_spend` actually contains utxos from the armada `rpc listunspent`. Assuming the miniwallet allows this tx to be created with utxos it cant sign, the return object contains a `"hex"` string that could be passed back to the armada node with `rpc signrawtransactionwithwallet`
